test/unittests/state_bloom_test.cpp

```diff
@@ -0,0 +1,88 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "../utils/utils.hpp"
+#include <evmc/evmc.hpp>
+#include <gtest/gtest.h>
+#include <test/statetest/statetest.hpp>
+
+using namespace evmc::literals;
+using namespace evmone::state;
+
+TEST(state_bloom, combine_blooms)
+{
+    // https://sepolia.etherscan.io/block/2946594
+
+    auto block_bloom =
```

Comment:
```suggestion
    constexpr auto block_bloom =
```

Reply:
`_hex` is not constexpr so only regular `const` is possible here.